### PR TITLE
Allow non-kingmaker worlds to enable kingdom and army sheets

### PR DESCRIPTION
**Pull request by CarlosFdez**
Supercedes https://github.com/foundryvtt/pf2e/pull/9922

### CODE REVIEW

File: src/module/system/settings/automation.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export class AutomationSettings extends SettingsMenuPF2e {
     protected static override get settings(): Record<ConfigPF2eListName, PartialSettingsData> {
         return {
             rulesBasedVision: {
+                prefix: "automation.",
```
Comment (stwlam) [inline src/module/system/settings/automation.ts:21]: Can there be a default prefix?

File: static/lang/en.json [hunk 2]
```diff
@@ -3184,6 +3184,7 @@
                     "Name": "Spell Traits"
                 },
                 "Tabs": {
+                    "Campaign": "Campaign Settings",
```
Comment (stwlam) [inline static/lang/en.json:3187]:
```suggestion
                    "Campaign": "Campaign",
```

File: static/lang/en.json [hunk 3]
```diff
@@ -3089,8 +3089,8 @@
                 "Name": "Campaign Feats"
             },
             "CampaignType": {
-                "Hint": "Extended campaign rule type for parties",
-                "Name": "Campaign Type",
+                "Hint": "Extended campaign rule support for campaigns such as kingmaker",
+                "Name": "Campaign Specific Features",
```
Comment (stwlam) [inline static/lang/en.json:3093]:
```suggestion
                "Hint": "Extended campaign rule support for campaigns such as Kingmaker",
                "Name": "Campaign-Specific Features",
```

File: src/module/system/settings/automation.ts (hunk 1 above)
Comment (CarlosFdez) [reply src/module/system/settings/automation.ts:21]: There can be, though given the number of places that have to prepend the prefix, as well as a helper, that seemed like the worse of two evils.

The other concern was how do we remove a prefix. Thinking now, `prefix: null` could work.

Comment (stwlam) [reply src/module/system/settings/automation.ts:21]: If it's worse with a default, then that's fine.

File: static/templates/system/settings/homebrew.hbs [hunk 4]
```diff
@@ -2,6 +2,7 @@
     <nav class="sheet-tabs tabs">
         <a class="item" data-tab="traits">{{localize "PF2E.SETTINGS.Homebrew.Tabs.Traits"}}</a>
         <a class="item" data-tab="languages">{{localize "PF2E.Actor.Creature.Language.Plural"}}</a>
+        <a class="item" data-tab="settings">{{localize "PF2E.SETTINGS.Homebrew.Tabs.Campaign"}}</a>
```
Comment (stwlam) [inline static/templates/system/settings/homebrew.hbs:5]:
```suggestion
        <a class="item" data-tab="campaign">{{localize "PF2E.SETTINGS.Homebrew.Tabs.Campaign"}}</a>
```

**Pull request merged**